spec/plaintime.html

```diff
@@ -610,11 +612,14 @@ <h1>ToTemporalTime ( _item_ [ , _overflow_ ] )</h1>
         1. If Type(_item_) is Object, then
           1. If _item_ has an [[InitializedTemporalTime]] internal slot, then
             1. Return _item_.
-          1. Let _calendar_ be ? Get(_item_, *"calendar"*).
-          1. If _calendar_ is not *undefined*, then
-            1. Set _calendar_ to ? ToTemporalCalendar(_calendar_).
-            1. If ? ToString(_calendar_) is not *"iso8601"*, then
-              1. Throw a *RangeError* exception.
+          1. If _item_ has an [[InitializedTemporalZonedDateTime]] internal slot, then
+            1. Let _instant_ be ? CreateTemporalInstant(_item_.[[Nanoseconds]]).
+            1. Set _item_ to ? BuiltinTimeZoneGetPlainDateTimeFor(_item_.[[TimeZone]], _instant_, _item_.[[Calendar]]).
+          1. If _item_ has an [[InitializedTemporalDateTime]] internal slot, then
```

Comment:
```suggestion
          1. Else if _item_ has an [[InitializedTemporalDateTime]] internal slot, then
```
since these paths should be mutually exclusive?

Reply:
This particular path is not mutually exclusive (in lines 615–617 _item_ is modified from a ZonedDateTime into a PlainDateTime and subsequently "falls through" into the PlainDateTime clause) but reading it back, maybe it's better to be more explicit and not write it this way, because I can see where it could be confusing.

Reply:
ah, i see what you mean. i definitely find it confusing to reuse variable names, both in the spec and in actual code :-)

Reply:
Assuming that you're OK with "If" and not "Else if" if the previous clause Returns, I've fixed this up everywhere it occurs.